app/models/user_access.rb

```diff
@@ -57,6 +57,22 @@ def accessible_organizations(action)
       .includes(facility_groups: :facilities)
   end
 
+  class UnsupportedAccessRequest < RuntimeError; end
+
+  # We determine a state's view_reports access via the district manager user access. Any other access requests for
+  # a state receive an error and are not supported.
```

Comment:
Would it be a bit clearer to word the first half as: 

Users can view reports for a state if they can manage any district within the state.